frontend/src/components/search/SearchFilters.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import clsx from "clsx";
+
+import { useState } from "react";
+
+import ContentDisplayToggle from "src/components/ContentDisplayToggle";
+import SearchFilterAccordion, {
+  SearchFilterConfiguration,
+} from "src/components/search/SearchFilterAccordion/SearchFilterAccordion";
+import SearchOpportunityStatus from "src/components/search/SearchOpportunityStatus";
+
+export default function SearchFilters({
```

Comment:
split out largely so that we can avoid making the full page component a client component when we add state for controlling the toggle